### PR TITLE
attachments

### PR DESCRIPTION
**Pull request by jsirianni**

### CODE REVIEW

File: servers/kamyshowdown/init.c [hunk 1]
```diff
@@ -61,6 +61,9 @@ class CustomMission: MissionServer
 		return m_player;
 	}
 
+	// 0 or less is always false
```
Comment (algchoo) [inline servers/kamyshowdown/init.c:64]: So I've updated the logic in a fix to the feature branch for the decide function. Will need to edit this file accordingly.

Comment (algchoo) [reply servers/kamyshowdown/init.c:64]: @jsirianni Actually we will just need to remove these comments. I updated the function to be a little more clear in the other PR

**Review by algchoo — CHANGES_REQUESTED** [review]
Other than needing to delete comments, I think we're good.

File: servers/kamyshowdown/init.c (hunk 1 above)
Comment (algchoo) [reply servers/kamyshowdown/init.c:64]: For clarity it's just lines 64, 65, & 66

**Comment by jsirianni** [comment]
Going to merge this anyway, and merge [your pr after](https://github.com/jsirianni/dayz_config/pull/12) (this pr is running on the server just fine)

**Pull request merged**